packages/regl-worldview/src/stories/GLText.stories.js

```diff
@@ -86,5 +86,51 @@ storiesOf("Worldview/GLText", module)
         </Container>
       );
     }
+    return <Example />;
+  })
+  .add("highlighted text", () => {
+    const Example = () => {
+      const [searchText, setSearchText] = useState("ello\nW");
+      const markers = textMarkers({ text: "Hello\nWorldview" }).map((marker) => {
+        if (!searchText) {
+          return marker;
+        }
+        const highlightedIndices = new Set();
+        let match;
+        let regex;
+        try {
+          regex = new RegExp(searchText, "ig");
+        } catch (e) {
+          return marker;
+        }
+        while ((match = regex.exec(marker.text)) !== null) {
+          Array.from(Array(searchText.length).keys()).forEach((i) => {
```

Comment:
TIL about this technique for getting a range 😮  But also, feel free to just use `_.range()` :)

Should probably use `match[0].length` instead though.